.gitignore

```diff
@@ -15,3 +15,4 @@ test/images/
 *.pot
 POTFILES*
 tmp/weblate-repo
+tmp/run
```

Comment:
How about we just ignore tmp/ entirely? 

After this lands, can you please apply this to starter-kit as well? Thanks!

Reply:
Sounds good to me!